files/zh-cn/glossary/forbidden_header_name/index.md

```diff
@@ -0,0 +1,46 @@
+---
+title: 禁止修改的消息头
+slug: Glossary/Forbidden_header_name
+tags:
+  - Fetch
+  - 术语表
+  - HTTP
+  - 消息头
+  - 禁止
+translation_of: Glossary/Forbidden_header_name
+---
+
+**禁止修改的消息头**指的是不能在代码中通过编程的方式进行修改的 [HTTP 消息头](/zh-CN/docs/Web/HTTP/Headers)，具体指 HTTP **请求**消息头（和 {{Glossary("Forbidden response header name", "禁止修改的响应消息头")}} 形成对比）。
+
+因为用户代理保留对此类消息头的完全控制，所以禁止修改他们。预留以 `Sec-` 开头的消息头用于创建新消息头，这些新消息头在那些使用 [Fetch](/en-US/docs/Web/API/Fetch_API) 允许开发人员控制消息头的 {{glossary("API","APIs")}}（例如：{{domxref("XMLHttpRequest")}}）中禁止修改。
```

Comment:
The link of `fetch` may be modified later.
https://github.com/mdn/content/issues/17281

Reply:
"APIs" ---> "API"